aries-site/src/examples/components/datatable/DataTableVerticalListExample.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button, 
+  DataTable,
+  Heading,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import ServerGroups from '../../../data/mockData/serverGroups.json';
+
+/* This function displays an arr's values in a vertical list within a table 
+   cell, hiding all values from view that exceed the given maximum list 
+   length until the "View More" button is clicked, at which point the 
+   button label changes to "View Less". */
+function ViewMore(arr, maxParam) {
+    const [viewMoreBool, setViewMoreBool] = useState(false);
+    return (
+        <Box align="end">
+            <Text truncate>{viewMoreBool ? arr.map(tmp => 
+                <Box>
+                    <Text>{tmp}</Text>
+                </Box>) : arr.map(tmp => 
+                <Box>
+                    <Text>{tmp}</Text>
+                </Box>).slice(0, maxParam)
+                }
+            </Text>
+            {arr.length > maxParam && (
+                <Button 
+                    label = {viewMoreBool ? 'View less'
+                            : `View more (${arr.length - maxParam})`}
+                    onClick = {() => setViewMoreBool(!viewMoreBool)} 
+                />
+            )}
+        </Box>
+    );
+}
+
+const data = ServerGroups.groups;
+const max = 3;
+const columns = [
+    {
+        property: 'name',
+        header: 'Name',
+        render: datum => <Text truncate>{datum.name}</Text>,
+    },
+    {
+        property: 'groupName',
+        header: 'Group Name',
+        render: datum => <Text truncate>{datum.groupName}</Text>,
+        sortable: true,
+    },
+    {
+        property: 'servers',
+        header: 'Servers',
+        align: 'start',
+        sortable: false,
+        render: datum => (ViewMore(datum.servers, max)),
+    },
+    {
+        property: 'status',
+        header: 'Status',
+        sortable: false,
+    },
+];
+
+// designSystemDemo is used for DS site only, can be removed in production.
+export const DataTableVerticalListExample = ({ designSystemDemo }) => {
+const size = React.useContext(ResponsiveContext);
+  return (
+    <>
+      <Heading
+        id="server-groups-heading"
```

Comment:
Small note, we need to make this `id` unique to this example since it appears on the same page as the other example. Can't remember if it's the same as the original example you started with? If so, maybe changing this to `server-groups-vertical-align` and adjusted the `aria-describedby` on DataTable below to match.